posthog-ai/src/anthropic/index.ts

```diff
@@ -72,7 +72,7 @@ export class WrappedMessages extends AnthropicOriginal.Messages {
       return parentPromise.then((value) => {
         const passThroughStream = new PassThrough({ objectMode: true })
         let accumulatedContent = ''
-        let usage: { inputTokens: number; outputTokens: number } = {
+        const usage: { inputTokens: number; outputTokens: number } = {
```

Comment:
this should have stayed let - its gets resigned under

Reply:
the linter did it automatically btw (eslint, prettier), but as I can see it, the usage props are resigned, but not the `usage` itself.
Happy to change it back but am I missing something?